lib/mozilla/diff/compare.js

```diff
@@ -112,7 +112,7 @@ function parseEntryListDiffElem(entrylist1, entrylist2, elem) {
 Comparer.prototype.compareEntries = function(entry1, entry2, entryId) {
   var entryDiff = new diff.EntryDiff(entryId);
 
-  var severity = this.config.checkMore;
+  var severity = 30;
```

Comment:
Since we're mostly dealing with warnings and errors for now, maybe set this to levels.WARNING (which equals 30) and I'll fix this whole severity thing next week?